### ISSUE TITLE
Could this be ported to vim?

### ISSUE THREAD

**Issue by pianocomposer321** [Arcath/file-templates#20]
I can't seem to find any plugin like this for Vim. Is there one, or could this be ported to vim/neovim?

**Comment by jjssto** [comment]
> I can't seem to find any plugin like this for Vim. Is there one, or could this be ported to vim/neovim?

@pianocomposer321 Something similar can be achieved with a few line of code in the vimrc/init.vim file. The function GetSkeleton(template) on line 69 in my config file opens a template (which is saved in ~/.config/nvim/templates). This function then gets called by autocmnd when a file with a corresponding file ending is opened (line 118-123).  https://github.com/jjssto/Config/blob/master/nvim/init.vim

**Comment by pianocomposer321** [comment]
@jjssto I apologize, I wasn't looking for a boilerplate _file_ but an entire boilerplate _file structure_, like what [this](https://github.com/cantonios/vscode-project-templates) VSCode extension does. I didn't realize when I created this issue that that is all that this atom plugin does.

Anyway, I've created my own plugin for vim that does what I want. Here's the repo if you're interested: https://github.com/pianocomposer321/project-templates.nvim
